ddsreplayer/test/blackbox/mcap/McapFileReadTest.cpp

```diff
@@ -102,29 +102,29 @@ TEST(McapFileReadTest, data_to_check)
     ASSERT_EQ(data.message_msg, "Hello World");
     ASSERT_EQ(data.min_index_msg, 0);
     ASSERT_EQ(data.max_index_msg, 10);
-    // hz ~ 200
+    // ms ~ 200
```

Comment:
Maybe 15ms can be reduced to something tighter, in the intraprocess case there should be almost no latency.

Reply:
We also discussed about getting rid of the transient local requirement, which might affect the measurement.
Instead, a wait_for_matched method could be added to the subscriber.